helm-chart/renku/values.yaml

```diff
@@ -242,15 +255,15 @@ keycloak:
   initRealm:
     image:
       repository: renku/init-realm
-      tag: "latest"
+      tag: '0.23.1-5.5b17747.n045.h3cb67ea5'
```

Comment:
Reset the tags to what they were before. Chartpress will set them to the right thing when we publish the helm chart.